recipe/pyproject.patch

```diff
@@ -0,0 +1,56 @@
+--- /dev/null
++++ pyproject.toml
+@@ -0,0 +1,53 @@
++# This pyproject.toml file is used to build the conda package only.
++
++[build-system]
++requires = ["setuptools"]
++build-backend = "setuptools.build_meta"
++
++[project]
++name = "rdkit"
++description = "A collection of chemoinformatics and machine-learning software written in C++ and Python"
++authors = [{ name = "Greg Landrum", email = "greg.landrum@gmail.com" }]
++readme = "README.md"
++version = "0.0.0"
++requires-python = ">=3.8"
++license = { text = "BSD-3-Clause" }
++classifiers = [
++    "Development Status :: 5 - Production/Stable",
++    "Intended Audience :: Developers",
++    "Intended Audience :: Healthcare Industry",
++    "Intended Audience :: Science/Research",
++    "Topic :: Scientific/Engineering :: Artificial Intelligence",
++    "Topic :: Scientific/Engineering :: Bio-Informatics",
++    "Topic :: Scientific/Engineering :: Information Analysis",
++    "Topic :: Scientific/Engineering :: Medical Science Apps.",
++    "Natural Language :: English",
++    "Operating System :: OS Independent",
++    "Programming Language :: Python",
++    "Programming Language :: Python :: 3",
++    "Programming Language :: Python :: 3.8",
```

Comment:
Any particular build/install is only going to support a single python version, not the full gamut. I also don't like the fact that we'll have to update this every time a new python version needs to be supported.

It seems like it would make sense to set the specific python version as part of the patching we're already doing to set the package version.

Reply:
Actually, I put thse metadata by taking inspiration from rdkit-pypi but in that context and since this pypi package will not get published but simply used to build the conda package, I don't think those are necessary.

So I removed the different Python versions and only kept `"Programming Language :: Python :: 3"`